app/views/dashboard/show.html.haml

```diff
@@ -1,7 +1,7 @@
 - title t('homepage.title')
 
-.alert.alert-dark.mb-0.text-center
-  We’re beyond proud to launch our <strong>first-ever</strong> Impact Report, which you can read here - <a href="https://impact-report-22.codebar.io/" class="text-dark" target="_blank"">impact-report-22.codebar.io</a>
+-# .alert.alert-dark.mb-0.text-center
```

Comment:
I just commented this out for now in case we want to put it back after the festival is over.